src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -840,54 +799,54 @@ SubCmdProgram::execute(const SubCmdOptions& _options) const
   if (deviceCollection.size() == 0) 
     throw std::runtime_error("No devices found.");
 
-  // Get the device
-  auto & workingDevice = deviceCollection[0];
-
-  // TODO: Added mutually exclusive code for image, update, and revert-to-golden action.
-
-  if (!image.empty()) {
-
-    // image is a sub-option of update
-    if (update.empty())
-      throw xrt_core::error("Usage: xbmgmt program --device='0000:00:00.0' --base --image='/path/to/flash_image' OR shell_name");
-
-    // We support up to 2 flash images 
-    if (image.size() == 1)
-      auto_flash(workingDevice, flashType, image.front());
-    
-    if (image.size() == 2) {
-      std::cout << "CAUTION! Force flashing the platform on the device without any checks." <<
-                   "Please make sure that the correct information is passed in." << std::endl;
-      auto image_paths = find_flash_image_paths(image);
-      update_shell(workingDevice->get_device_id(), flashType, image_paths.front(), (image_paths.size() == 2 ? image_paths[1]: ""));
-    }
-    
-    if (image.size() > 2)
-      throw xrt_core::error("Please specify either 1 or 2 flash images");
-    return;
-  }
+  // Only two images options are supported
+  if (image.size() > 2)
+    throw xrt_core::error("Please specify either 1 or 2 flash images");
```

Comment:
Lets update this message as well.  Something like,
```suggestion
    throw xrt_core::error("Multiple flash images provided.  Please specify either 1 or 2 flash images.");
```